lib/components/SLocalNav.vue

```diff
@@ -0,0 +1,39 @@
+<script setup lang="ts">
+import SLocalNavMenu, { type MenuItem } from './SLocalNavMenu.vue'
+import SLocalNavTitle, { type Title } from './SLocalNavTitle.vue'
+
+export type { Title, MenuItem }
+
+defineProps<{
+  title: Title[]
+  menu?: MenuItem[][]
+}>()
+</script>
+
+<template>
+  <div class="SLocalNav">
+    <SLocalNavTitle :title="title" />
+    <SLocalNavMenu v-if="menu" :menu="menu" />
+  </div>
+</template>
+
+<style scoped lang="postcss">
+.SLocalNav {
+  display: flex;
+  flex-direction: column;
+  gap: 8px;
+  padding: 16px 24px;
+
+  @media (min-width: 768px) {
+    padding: 16px 32px;
+  }
+}
+
+.SLocalNav:has(.SLocalNavMenu) {
```

Comment:
This is new. Won't work with anything released before last year 😅

Reply:
Ah, I thought this one was way older 😓 I'll change it.

> Though, managing active is left to the component using this. Not sure how complex that will be 👀

It will add active status using `router-link-active` too. So if we set correct link, it should be automatic.

The `active` option was for more like demonstration purpose on Histoire and for escape hatch 👍

Reply:
Fixed!